crypto/bio/b_addr.c

```diff
@@ -603,7 +604,7 @@ static int addrinfo_wrap(int family, int socktype,
 DEFINE_RUN_ONCE_STATIC(do_bio_lookup_init)
 {
     OPENSSL_init_crypto(0, NULL);
-    bio_lookup_lock = CRYPTO_THREAD_glock_new("bio_lookup");
+    bio_lookup_lock = global_locks[CRYPTO_GLOCK_BIO_LOOKUP];
```

Comment:
Why not just make bio_lookup_lock (and similar for others) be a define?
Avoid the extra package-level global variable.

Reply:
I had been considering doing a search-and-replace as a follow-up commit (but leaving the first one easier to review by keeping the churn out).  You think the macro is better?

Reply:
Either way's fine with me.  I think a #define will be easier, but leave it up to you.  But I think it should be part of this PR, although as a separate commit.

Reply:
I think the define would be nicer too but not really fussed either way.  If a macro is used, it should be done in this PR.